doc/topics/commands.rst

```diff
@@ -605,8 +605,9 @@ The music database
 
     *URI* is always a single file or URL.
 
-    The returned value is the binary contents of the album art image. If the given URI is invalid,
-    or the song does not have an album cover art file that MPD recognizes, a CommandError is thrown.
+    The returned value is a dictionary containing the album art image in its
+    ``'binary'`` entry. If the given URI is invalid, or the song does not have
+    an album cover art file that MPD recognizes, a CommandError is thrown.
```

Comment:
So albumart will throw an error if a song has no picture or only if it is invalid?

Reply:
MPD will report an error if no album art is found (and it does not do any validation on it). This is inconsistent with its handling of readpicture, which returns empty OK if there's no picture.

Given that most of python-mpd2 is diving comparatively raw access to most MPD functionality, it may make sense to accept this inconsistency here, but about it in the documentation and possibly ask for clarification in the protocol development.

Reply:
ok.